public/app/plugins/panel/geomap/editor/MapViewEditor.tsx

```diff
@@ -112,6 +112,37 @@ export const MapViewEditor: FC<StandardEditorProps<MapViewConfig, any, GeomapPan
         </InlineField>
       </InlineFieldRow>
 
+      {value?.id === MapCenterID.Fit && (
+        <>
```

Comment:
This would probably make sense to split out into an individual component for improved maintainability / testing

Reply:
Will look into that after resolving conflicts on GeomapPanel.